version.json

```diff
@@ -1,4 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "17.4"
+  "version": "17.4",
+  "cloudBuild": {
+    "setAllVariables": true
```

Comment:
We end up using the `NBGV_NuGetPackageVersion` value as the PackageVersion for VS insertion.